Dockerfile

```diff
@@ -1,3 +1,6 @@
+# URGENT! ART metadata configuration has a different number of FROMs
+# than this Dockerfile. ART will be unable to build your component or
+# reconcile this Dockerfile until that disparity is addressed.
```

Comment:
@jupierce @sosiouxme Is this comment accurate since we use Dockerfile.product for product builds? (That has changed in master with #12444)